src/main/java/samza/examples/wikipedia/application/WikipediaZkLocalApplication.java

```diff
@@ -32,7 +32,15 @@
  */
 public class WikipediaZkLocalApplication {
 
-  public static void main(String[] args) throws Exception {
+  /**
+   * Executes the application using the local application runner.
+   * It takes two required command line arguments
+   *  config-factory: a fully {@link org.apache.samza.config.factories.PropertiesConfigFactory} class name
+   *  config-path: path to application properties
+   *
+   * @param args command line arguments
+   */
+  public static void main(String[] args) {
```

Comment:
I see your point here and the pattern suggested. For this scenario, it probably doesn't make any difference since the entry point is specific to wikipedia application. IMO, its okay to be liberal because it gives us the flexibility to perform any sort of app specific setup if needed prior to execution.